### ISSUE TITLE
Correct gebruik attributes

### ISSUE THREAD

**Issue by hphoeksma** [repeeter1/2.01-simple-portfolio#4]
https://github.com/repeeter1/2.01-simple-portfolio/blob/a1e3d53cbd7408bf28f92c7ab92fa8127a0d1e3a/index.html#L55

Je typed hier een value in de textarea. Waarom gebruik je geen placeholder attribuut?

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea

**Comment by repeeter1** [comment]
daar heb je een punt. ik zal het aanpassen

**Comment by hphoeksma** [comment]
Maar wat is mijn punt dan? (Ik zoek naar je gedachte achter je oplossing)

**Comment by repeeter1** [comment]
Er zit geen gedachten achter, ik ben vergeten een placeholder te gebruiken. Hoe ik het nu gedaan heb, blijft de tekst ook gewoon staan en gedraagt hij zich ook niet als een placeholder.

**Comment by hphoeksma** [comment]
Juist. Placeholder is een HTML5 dingetje en wordt wel vaker vergeten. Vandaar dat ik hem nog ff onder de aandacht breng!
